src/core/public/public.api.md

```diff
@@ -460,56 +455,33 @@ export interface HttpHeadersInit {
     [name: string]: any;
 }
 
-// Warning: (ae-missing-release-tag) "HttpInterceptController" is exported by the package, but it is missing a release tag (@alpha, @beta, @public, or @internal)
-// 
-// @public (undocumented)
-export class HttpInterceptController {
-    // (undocumented)
-    halt(): void;
-    // (undocumented)
-    readonly halted: boolean;
-    }
-
-// @public (undocumented)
+// @public
 export interface HttpInterceptor {
-    // (undocumented)
-    request?(request: Request, controller: HttpInterceptController): Promise<Request> | Request | void;
-    // (undocumented)
-    requestError?(httpErrorRequest: HttpErrorRequest, controller: HttpInterceptController): Promise<Request> | Request | void;
-    // (undocumented)
-    response?(httpResponse: HttpResponse, controller: HttpInterceptController): Promise<HttpResponse> | HttpResponse | void;
-    // (undocumented)
-    responseError?(httpErrorResponse: HttpErrorResponse, controller: HttpInterceptController): Promise<HttpResponse> | HttpResponse | void;
+    request?(request: Request, controller: IHttpInterceptController): Promise<Request> | Request | void;
+    requestError?(httpErrorRequest: HttpErrorRequest, controller: IHttpInterceptController): Promise<Request> | Request | void;
+    response?(httpResponse: HttpResponse, controller: IHttpInterceptController): Promise<HttpResponse> | HttpResponse | void;
+    responseError?(httpErrorResponse: HttpErrorResponse, controller: IHttpInterceptController): Promise<HttpResponse> | HttpResponse | void;
 }
 
-// @public (undocumented)
+// @public
 export interface HttpRequestInit {
+    // Warning: (ae-unresolved-inheritdoc-reference) The @inheritDoc reference could not be resolved: The package "kibana" does not have an export "RequestInit"

```

Comment:
https://github.com/elastic/kibana/pull/47069/files#diff-d1fdf4d9c9d69c4fdfc4b6963e9050cfR115 is that referencing a type that doesn't exist or are we just not exporting it?

Reply:
This is a type from TypeScript's DOM types (RequestInit is part of the browser's Fetch API). I'll just copy in the doc directly for now.